controllers/replica_set_controller.go

```diff
@@ -218,6 +228,10 @@ func (r ReplicaSetReconciler) Reconcile(ctx context.Context, request reconcile.R
 		return res, err
 	}
 
+	if err := r.updateConnectionStringSecrets(mdb); err != nil {
+		r.log.Errorf("Could not update connection string secrets: %s", err)
```

Comment:
If I remember correctly, error messages should not start with an uppercase letter

Reply:
this is a log and there are many (10+) similar error logs in replica_set_controller.go
I can update all to use the lowercase format if this is part of our style guide

Reply:
Maybe let's leave it for next time :) I think this is more for when returning errors